### PR TITLE
Add basic README

### PR DESCRIPTION
**Pull request by Monadic-Cat**
This project is overdue for a basic run down of making it work.
So, here we are.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+```sh
+git clone https://github.com/cbpudding/link-all-the-languages
+cd link-all-the-languages
+./run.sh
+```
+
+## Languages Used (so far)
+ - [C](https://en.wikipedia.org/wiki/C_(programming_language))
+ - [C++](https://isocpp.org/)
+ - [D (DMD)](https://dlang.org/)
```
Comment (cbpudding) [inline README.md]: Carp should be between these two

File: README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ - [Rust (Cargo)](https://www.rust-lang.org/)
+ - [Fortran (GNU)](https://gcc.gnu.org/fortran/)
+ - [Zig](https://ziglang.org/)
+ - [Carp](https://github.com/carp-lang/Carp)
```
Comment (cbpudding) [inline README.md]: Not here

File: README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# Link All The Languages!
+-------------------------
+
+A big bundle of different languages all being invoked from a C program.
+
+## Why?
+
+We want to see if we can!
```
Comment (cbpudding) [inline README.md:8]: The idea is that if we can link with C in all of these languages, we can link any of these languages in any order even without C. Not that this statement is inaccurate.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+## Getting Started
+This project has two build systems:
+ - A GNU Makefile
+ - A custom Python 3 script
+If you choose to use the GNU Makefile, you must install all of the language toolchains (listed below)
+used in this project.
```
Comment (cbpudding) [inline README.md]: I'd actually like to start deprecating the Makefile in favor of your Python script.

**Review by cbpudding — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -7,6 +7,9 @@ A big bundle of different languages all being invoked from a C program.
 
 We want to see if we can!
 
+The idea here is that if we can link all these languages into C,
+we can link them in any order without C.
```
Comment (cbpudding) [inline README.md]: With or without but okay

File: README.md (hunk 1 above)
Comment (Monadic-Cat) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (Monadic-Cat) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (Monadic-Cat) [reply README.md:8]: Done.

File: README.md (hunk 4 above)
Comment (Monadic-Cat) [reply README.md]: I've made a [tracking issue](https://github.com/cbpudding/link-all-the-languages/issues/13) for this. Added notice about that in the README.

File: README.md (hunk 5 above)
Comment (Monadic-Cat) [reply README.md]: lol i'll fix it

**Review by cbpudding — APPROVED** [review]
Looks good.

**Pull request merged**